### DISCUSSION TITLE
Metabolite Nomenclature

### DISCUSSION THREAD

**Discussion by Berat-Sahin** [SysBioChalmers/Human-GEM#246] [Q&A] [answered]
Hello,
I am trying to do an FBA in HumanGEM but the metabolites and constraints I am planning to use will be transferred from another manually constructed GEM. 

Sometimes the names of the metabolites don't particularly match since there might be different nomenclatures for one specific metabolite. So far, I was checking each metabolite by hand but it has become very time-consuming and I might be skipping some even if it is defined. 

My question is that do you have any sheet or a script that can help me somehow? Or any method that I can incorporate into my workflow?

Also, it is great what your group is doing. 

Thank you

**Comment by haowang-bioinfo** [comment, marked as answer]
@Berat-Sahin if this [table](https://github.com/SysBioChalmers/Human-GEM/blob/master/model/metabolites.tsv) is what you are looking for?

**Reply by Berat-Sahin** [reply]
Thank you very much. It was exactly what I was in need of.
